addons/docs/src/frameworks/react/lib/inspection/acornParser.ts

```diff
@@ -24,6 +24,7 @@ interface ParsingResult<T> {
 }
 
 const ACORN_WALK_VISITORS = {
```

Comment:
@shilman I assume the typings improved upstream from any to something specific, which might still be incomplete, OR we're doing something wrong perhaps?

I added ignores, because i assume it works as expected (test will tell?)